packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -22,7 +22,7 @@ import '../sub-components/mgt-flyout/mgt-flyout';
 import { MgtFlyout } from '../sub-components/mgt-flyout/mgt-flyout';
 import { PersonCardInteraction } from './../PersonCardInteraction';
 import { styles } from './mgt-person-css';
-import { Presence } from '@microsoft/microsoft-graph-types-beta';
+import { IdentitySourceType, Presence } from '@microsoft/microsoft-graph-types-beta';
```

Comment:
I don't think you need `IdentitySourceType` here. 

I believe Presence is now available as part of the v1 types - try to see if you can import Presence from the the `@microsoft/microsoft-graph-types` package instead

Reply:
Yup, seems to work great, let me know you'd rather the other import sytax though:

f1e826c